### PR TITLE
 test exchange bugfix, and simple_app logistics 

### PR DESCRIPTION
**Pull request by nasaWelder**
merkato will not make bids from gui unless you set that conditional to "if True:"

### CODE REVIEW

File: merkato/utils/database_utils.py [hunk 1]
```diff
@@ -151,7 +157,11 @@ def merkato_exists(UUID):
     finally:
         c = conn.cursor()
         c.execute('''SELECT * FROM merkatos WHERE exchange_pair = ?''', (UUID,))
-        result = len(c.fetchall())
+        query = c.fetchall()
```
Comment (mark-antony1) [inline merkato/utils/database_utils.py]: what is currently wrong with this? The print statements are not necessary, and the function works as is to my knowledge.

Comment (needmoney90) [reply merkato/utils/database_utils.py]: I'll wait on a response before merging

Comment (nasaWelder) [reply merkato/utils/database_utils.py]: that's not a bugfix, I was trying do debug why a GUI merkato thinks it doesn't need to distribute ladders. If you run the simple app and try to make a test exchange merkato, those print statements would show you it thinking the merkato exists already.

Comment (mark-antony1) [reply merkato/utils/database_utils.py]: merkato_exists should be returned to its state prior to this commit

**Comment by needmoney90** [comment]
Will approve once merkato_exists is reverted to the previous version

**Comment by nasaWelder** [comment]
@livinginformation  reverted it.

**Pull request merged**